src/appleseed/renderer/modeling/entity/entityfactoryregistrar.h

```diff
@@ -50,13 +59,21 @@ namespace renderer      { class Plugin; }
 namespace renderer
 {
 
+//define loaded libraries container structure
+typedef  std::vector<std::pair<std::unique_ptr<foundation::SharedLibrary>, std::string> >  loaded_libs_container;
```

Comment:
As per our [naming conventions](https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines#naming-conventions) (please read them) that should be `LoadedLibsContainer`.

Also note that in C++11, you no longer need to insert spaces between `>`.

Finally I think the comment doesn't add much and could be removed.